tests/flexalloc_ut_slab.c

```diff
@@ -22,12 +22,13 @@ int
 main(int argc, char ** argv)
 {
   int err = 0;
+  bool fla_test_set = is_globalenv_set("FLA_TEST_DEV");
   struct test_vals test_vals [FLA_UT_SLAB_NUMBER_OF_TESTS] =
   {
-    {.npools = 2, .slab_nlb = 2, .disk_min_lbs = 9, .obj_nlb = 1 }
-    , {.npools = 2, .slab_nlb = 2, .disk_min_lbs = 21, .obj_nlb = 1 }
-    , {.npools = 2, .slab_nlb = 20, .disk_min_lbs = 50, .obj_nlb = 2 }
-    , {.npools = 2, .slab_nlb = 5, .disk_min_lbs = 18, .obj_nlb = 1 }
+    {.npools = 2, .slab_nlb = 2, .disk_min_lbs = fla_test_set?0:9, .obj_nlb = 1 }
```

Comment:
I was went back and forth on this. I ended up in the ternary because it was less lines of code, but I can think on an additional way of doing it by introducing an if statement just before we call test_slabs.